#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_gui.dpatch by Jelmer Vernooij <jelmer@samba.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable UI building

@DPATCH@
diff -ur nunit-2.4.7/src/nunit.build.include nunit-2.4.7-noui/src/nunit.build.include
--- nunit-2.4.7/src/nunit.build.include	2008-03-30 08:44:12.000000000 +0200
+++ nunit-2.4.7-noui/src/nunit.build.include	2008-05-27 01:29:58.000000000 +0200
@@ -368,7 +368,7 @@
     <property name="runtime.defines" value="MONO,MONO_1_0"/>
     <property name="build.mono" value="true"/>
     <property name="build.win32" value="false"/>
-    <property name="build.gui" value="true"/>
+    <property name="build.gui" value="false"/>
     <property name="build.x86" value="false"/>
     <property name="nant.settings.currentframework" 
       value="mono-1.0"/>
@@ -389,7 +389,7 @@
     <property name="runtime.defines" value="MONO,MONO_2_0"/>
     <property name="build.mono" value="true"/>
     <property name="build.win32" value="false"/>
-    <property name="build.gui" value="true"/>
+    <property name="build.gui" value="false"/>
     <property name="build.x86" value="true"/>
     <property name="nant.settings.currentframework" 
       value="mono-2.0"/>
